### ISSUE TITLE
A11y_ML .NET Model Builder_Object Detection-Evaluate_Keyboard: User is unable to enter score threshold value manually using keyboard in 'Score threshold' field.

### ISSUE THREAD

**Issue by NiharikaTipirishetty** [dotnet/machinelearning-modelbuilder#1660]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #BM-VisualStudioClient-Win32-Jul21; #A11ySev2; #Visual Studio Client; #WCAG2.1.1; #Keyboard; #A11yPreview; 

### Environment Details:
Microsoft Visual Studio

### Re-pro Steps:

1. Turn on Narrator/NVDA. Open Visual Studio Enterprise 2022 Preview
2. Create a new .Net Core Console App (C#) ex: WpfApp
3. Tab to "Solution Explorer" and right click on the project (Shift + F10) and then from the expanded list Select "Add" and then from the list items select "Machine learning"
4. Tab till "Object Detection" and press enter
5. Select "Azure" environment. Tab till "Setup Work Space" button and press enter and create a new experiment and tab till "next" button and press enter
6. Select input data folder and tab till "Next" button and press enter
7. Tab till "Train" button and press enter. After training is completed, navigate to evaluate tab.
8. In Evaluate tab, upload a picture in try your model section and navigate to 'Score threshold' value field
9. Enter decimal values in the field through keyboard and verify user is able to enter decimal values or not

### Actual Result:
User is unable to enter score threshold value(decimal values) manually using keyboard in 'Score threshold' field.

### Expected Result:
User should be able to enter score threshold value(decimal values) manually using keyboard in 'Score threshold' field.

### User Impact:
Keyboard only users will find it difficult to increase/decrease threshold value if its not keyboard accessible

**Comment by beccamc** [comment]
Validate on  rel/candidate build 16.7.2.2141903. See issue 1627 for instructions for install.

**Comment by beccamc** [comment]
Here is a link the most recent Model Builder build. I’ve tested it on VS 17.0.0 Preview 5 

Model Builder: https://devdiv.visualstudio.com/_apis/resources/Containers/8681620/VSSetup_Release17?itemPath=VSSetup_Release17%2FMicrosoft.ML.ModelBuilder.Gallery.Dev17.vsix
VS: https://aka.ms/vs/17/pre/vs_Enterprise.exe

**Comment by NiharikaTipirishetty** [comment]
Issue is fixed. User is able to enter score threshold value manually using keyboard. Hence issue is fixed.


https://user-images.githubusercontent.com/83570638/133390393-c4667472-2b3d-4888-8384-fd434ac393fb.mp4

**Comment by akashelijah05** [comment]
#Closed;
